### PR TITLE
feat(wsctl): add project configuration cmd

### PR DESCRIPTION
**Pull request by hunshenshi**
# Description
Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

1. delete code cmd  in wsctl
2. add project configuration cmd in wsctl

## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Code refactor or improvement
- [] Breaking change (fix or feature that would cause a new or changed behavior of existing functionality)
- [] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [] make test
- [] fullsync
- [] Other test (please specify)

**Test Configuration**:
- Firmware version:
- Hardware:
- Toolchain:
- SDK:

# Checklist:
- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes
- [] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ioctl/cmd/ws/wsprojectconfig.go [hunk 1]
```diff
@@ -17,10 +17,10 @@ import (
 )
 
 var (
-	// wsCodeConvert represents the w3bstream code convert command
-	wsCodeConvert = &cobra.Command{
-		Use:   "convert",
-		Short: config.TranslateInLang(wsCodeConvertShorts, config.UILanguage),
+	// wsProjectConfig represents the generate w3bstream project configuration command
+	wsProjectConfig = &cobra.Command{
+		Use:   "configuration",
```
Comment (Liuhaai) [inline ioctl/cmd/ws/wsprojectconfig.go:22]: config

File: ioctl/cmd/ws/wsprojectconfig.go [hunk 2]
```diff
@@ -42,8 +42,12 @@ var (
 			if err != nil {
 				return errors.Wrap(err, "failed to get flag expand-param")
 			}
+			outputFile, err := cmd.Flags().GetString("output-file")
```
Comment (Liuhaai) [inline ioctl/cmd/ws/wsprojectconfig.go:45]: will err be returned without output flag?

File: ioctl/cmd/ws/wsprojectconfig.go [hunk 3]
```diff
@@ -147,3 +173,28 @@ func convertCodeToZlibHex(codeFile string) (string, error) {
...
+type vmType string
+
+const (
+	risc0  vmType = "risc0"  // risc0 vm
+	halo2  vmType = "halo2"  // halo2 vm
+	zkWasm vmType = "zkwasm" // zkwasm vm
+	wasm   vmType = "wasm"   // wasm vm
+)
+
+func stringToVMType(vmType string) (vmType, error) {
```
Comment (Liuhaai) [inline ioctl/cmd/ws/wsprojectconfig.go:187]: this should be/has been done at API side

Comment (hunshenshi) [reply ioctl/cmd/ws/wsprojectconfig.go:188]: add TODO

File: ioctl/cmd/ws/wsprojectconfig.go (hunk 2 above)
Comment (hunshenshi) [reply ioctl/cmd/ws/wsprojectconfig.go:45]: keep same with other param's error

**Review by Liuhaai — APPROVED** [review]

**Review by huangzhiran — APPROVED** [review]

**Pull request merged**